### PR TITLE
Kevinf bamboo

### PR DESCRIPTION
**Pull request by kfansler**
Mostly based on the Jupiter spec. I started this branch so long ago that there *might* be other minor changes.

### CODE REVIEW

File: source/_reference-t-z.html.md.erb [hunk 1]
```diff
@@ -2097,7 +2094,7 @@ Gets the specified workspace (and lists its contents).
 -----|-----|
 <%= HEADERS %> | <%= HEADER_AUTH %>
 <%= PARAMETERS %> | **loadAll** (optional) -- **true** or **false** (defaults to **false**)
-	| **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**ownerInfo** - owner's email address and user Id for each sheet</li><li>**source** - the [Source object](#source-object) for any sheet that was created from another sheet or template</li></ul>
+	| **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**ownerInfo** - owner's email address and user Id for each sheet</li><li>**source** - the [Source object](#source-object) for any sheet that was created from another report, sheet, Sight (aka dashboard), or template, if any</li></ul>
```
Comment (jeoff-smartsheet) [inline source/_reference-t-z.html.md.erb:2097]: Is there a reason `Sight` is capitalized but report and sheet are not?

File: source/_reference-g-s.html.md.erb [hunk 2]
```diff
@@ -7702,20 +7740,32 @@ Web Content | **WEBCONTENT** | [WebContentWidgetContent](#webcontentwidgetconten
...
 **htmlContent** | string | HTML snippet to render report
+**hyperlink** | [Hyperlink](#hyperlink-object) | The widget has **when clicked** attribute set to that hyperlink (if present and non-null). Hyperlinks will have **interactionType**.
 
 ### RichTextWidgetContent Object
 
      |     |     |
 -----|-----|-----|
 **html** | string | The widget content as HTML </br> </br> The Rich Text widget supports the following subset of HTML tags and CSS Styles: </br></br> <b>HTML</b><ul><li>**a** - defines a hyperlink</li> <li>**br** - inserts a single line break</li><li>**li** - defines a list item</li> <li>**ol** - defines an ordered list</li> <li>**p** - defines a paragraph</li> <li>**ul** - defines an unordered list</li> <li>**span** - defines a section in a document</li></ul> <b>CSS</b><ul><li>**color**</li> <li>**font-family**</li> <li>**font-size**</li> <li>**font-style**</li> <li>**font-weight**</li> <li>**text-align**</li> <li>**text-decoration**</li></ul>
 
+### SelectionRanges Object
```
Comment (jeoff-smartsheet) [inline source/_reference-g-s.html.md.erb]: Would it be more accurate for us to call this object `SelectionRange` vs the plural version. At least for me I imagine the plural being used for an array that contains many `SelectionRange` objects

File: source/_reference-g-s.html.md.erb [hunk 3]
```diff
@@ -863,7 +863,7 @@ Gets a nested list of all Home objects, including folders, reports, sheets, Sigh
      |     |
 -----|-----|
 <%= HEADERS %> | <%= HEADER_AUTH %>
-<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**source** - the [Source object](#source-object) for any sheet that was created from another sheet or template</li></ul>
+<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**source** - the [Source object](#source-object) for any sheet that was created from another report, sheet, Sight (aka dashboard), or template, if any</li></ul>
```
Comment (jeoff-smartsheet) [inline source/_reference-g-s.html.md.erb:866]: Similar comment like below asking about the capitalization of `Sight`

File: source/_reference-a-f.html.md.erb [hunk 4]
```diff
@@ -7102,7 +7134,7 @@ Gets the specified folder (and lists its contents).
      |     |
 -----|-----|
 <%= HEADERS %> | <%= HEADER_AUTH %>
-<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**ownerInfo** - owner's email address and user Id for each sheet</li><li>**sheetVersion** - current version number of each sheet, should not be combined with pagination</li><li>**source** - the [Source object](#source-object) for any sheet that was created from another sheet or template</li></ul>
+<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**ownerInfo** - owner's email address and user Id for each sheet</li><li>**sheetVersion** - current version number of each sheet, should not be combined with pagination</li><li>**source** - the [Source object](#source-object) for any sheet that was created from another report, sheet, Sight (aka dashboard), or template, if any</li></ul>
```
Comment (jeoff-smartsheet) [inline source/_reference-a-f.html.md.erb:7137]: Similar comment like below asking about the capitalization of `Sight`

File: source/_reference-g-s.html.md.erb [hunk 5]
```diff
@@ -1488,7 +1488,7 @@ Gets the report, including one page of rows, and optionally populated with attac
      |     |
 -----|-----|
 <%= HEADERS %> | <%= HEADER_AUTH %>
-<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**attachments**</li><li>**discussions**</li><li>**format** -- see [Formatting](#formatting)</li><li>**objectValue** -- when used in combination with a **level** query parameter, includes the email addresses for multi-contact data</li><li>**sourceSheets**</li></ul> 
+<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**attachments**</li><li>**discussions**</li><li>**format** -- see [Formatting](#formatting)</li><li>**objectValue** -- when used in combination with a **level** query parameter, includes the email addresses for multi-contact data</li><li>**scope** -- adds the report scope to the response</li><li>**source** -- adds the [Source object](#source-object) indicating which report the report was created from, if any</li><li>**sourceSheets**</li></ul>
```
Comment (jeoff-smartsheet) [inline source/_reference-g-s.html.md.erb]: After our discussion I believe we want to remove the reference to scope

File: source/_reference-g-s.html.md.erb [hunk 6]
```diff
@@ -5189,7 +5189,7 @@ A sheet can exist in a user's **Sheets** folder ([Home](#home)), in a [folder](#
 **resourceManagementEnabled** | Boolean | Indicates that resource management is enabled
 **rows** | [Row](#row-object)[] | Array of Row objects
 **showParentRowsForFilters** | Boolean | Returned only if there are column filters on the sheet.  Value = **true** if "show parent rows" is enabled for the filters.
-**source** | [Source](#source-object) | A Source object indicating the sheet or template from which this sheet was created
+**source** | [Source](#source-object) | A Source object indicating the report, sheet, Sight (aka dashboard), or template from which this sheet was created, if any
```
Comment (jeoff-smartsheet) [inline source/_reference-g-s.html.md.erb:5192]: Similar comment like below asking about the capitalization of `Sight`

File: source/_reference-g-s.html.md.erb [hunk 7]
```diff
@@ -5300,8 +5300,8 @@ Represents individual user settings for a specific sheet.  User settings may be
 
      |     |     |
 -----|-----|-----|
-**id** | number | Id of the sheet or template from which the enclosing sheet was created
-**type** | string | **sheet** or **template**
+**id** | number | Id of the report, sheet, Sight (aka dashboard), or template from which the enclosing report, sheet, Sight, or template was created
```
Comment (jeoff-smartsheet) [inline source/_reference-g-s.html.md.erb:5303]: Similar comment like below asking about the capitalization of `Sight`

File: source/_reference-g-s.html.md.erb [hunk 8]
```diff
@@ -6545,7 +6545,7 @@ Gets the sheet specified in the URL. Returns the sheet, including rows, and opti
...
+<%= PARAMETERS %> | **include** (optional): a comma-separated list of elements to include in the response.<br/>Supports all [Row Include Flags](#row-include-flags) with the following differences:<ul><li>**attachments**: includes the metadata for sheet-level and row-level attachments</li><li>**crossSheetReferences**: includes the cross-sheet references</li><li>**discussions**: includes sheet-level and row-level discussions</li><li>**filters**: includes [row.filteredOut](#row-object) attribute</li><li>**filterDefinitions**: includes type of filter, operators used, and criteria</li><li>**format**: includes column, row, and cell formatting</li><li>**ganttConfig**: includes Gantt chart details</li><li>**objectValue** -- when used in combination with a **level** query parameter, includes the email addresses for multi-contact data</li><li>**ownerInfo**: includes the workspace and the owner's email address and user Id</li></ul>Also supports the **source** include flag, which adds the [Source object](#source-object) indicating which report, sheet, Sight (aka dashboard), or template the sheet was created from, if any.
```
Comment (jeoff-smartsheet) [inline source/_reference-g-s.html.md.erb:6548]: Similar comment like below asking about the capitalization of `Sight`

File: source/_reference-g-s.html.md.erb [hunk 9]
```diff
@@ -6941,7 +6941,7 @@ Gets a list of all sheets that the user has access to in alphabetical order by n
      |     |
 -----|-----|
 <%= HEADERS %> | <%= HEADER_AUTH %>
-<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**sheetVersion** - current version number of each sheet, should not be combined with pagination</li><li>**source** - the [Source object](#source-object) for any sheet that was created from another sheet or template</li></ul>
+<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of optional elements to include in the response:<ul><li>**sheetVersion** - current version number of each sheet, should not be combined with pagination</li><li>**source** - the [Source object](#source-object) for any report, sheet, Sight (aka dashboard), or template that was created from another report, sheet, Sight, or template, if any</li></ul>
```
Comment (jeoff-smartsheet) [inline source/_reference-g-s.html.md.erb:6944]: Similar comment like below asking about the capitalization of `Sight`

**Review by jeoff-smartsheet — APPROVED** [review]
Other than the comments I made, looks good to me
There are many spots where I made the same comment, just to be safe but if `Sight` is intentionally capitalized then those can be resolved

File: source/_reference-t-z.html.md.erb (hunk 1 above)
Comment (kfansler) [reply source/_reference-t-z.html.md.erb:2097]: Yes, Sight is a Smartsheet UI term and capitalized in the official corporate style guide.

**Pull request merged**